lib/googlestaticmap.rb

```diff
@@ -119,14 +123,15 @@ def url(protocol='http')
     attrs = GoogleStaticMapHelpers.safe_instance_variables(self,
               ["markers", "paths", "width", "height", "center",
                "proxy_address", "proxy_port", "api_key", "client_id",
-               "private_key"],
+               "private_key", "language"],
               :cgi_escape_values => true).to_a
     attrs << ["size", "#{@width}x#{@height}"] if @width && @height
     @markers.each {|m| attrs << ["markers",m.to_s] }
     @paths.each {|p| attrs << ["path",p.to_s] }
     attrs << ["center", @center.to_s] if !@center.nil?
     attrs << ["key", @api_key] if !@api_key.nil?
     attrs << ["client", @client_id] if !@client_id.nil?
+    attrs << ["language", @language] if !@language.nil?
```

Comment:
So in other words, by adding language as an attr_accessor, it will be added to the request with no additional code, it will work just like format, scale, etc